pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -17,13 +17,36 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"time"
+
 	ndmapis "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	blockdevice "github.com/openebs/maya/pkg/blockdevice/v1alpha1"
+	bdc "github.com/openebs/maya/pkg/blockdeviceclaim/v1alpha1"
+	env "github.com/openebs/maya/pkg/env/v1alpha1"
+	volume "github.com/openebs/maya/pkg/volume"
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+	retryCount = 10
+	waitTime   = 5 * time.Second
+)
+
+// ClaimedBDDetils holds the claimed block device details
+type ClaimedBDDetails struct {
+	DeviceID string
+	BDName   string
+}
+
+// NodeClaimedBDDetails holds the node name and
+// claimed block device deatils corresponding to node
+type NodeClaimedBDDetails struct {
+	NodeName        string
+	BlockDeviceList []ClaimedBDDetails
+}
+
 // NodeBlockDeviceSelector selects a node and disks attached to it.
```

Comment:
Because the function has changed, better to update the comment too.

Reply:
Updated